pyrav4l2/device.py

```diff
@@ -50,14 +50,26 @@ def __init__(self, numerator: int = 0, denominator: int = 0) -> None:
         self.numerator = numerator
         self.denominator = denominator
 
+    def __eq__(self, other: FrameInterval) -> bool:
+        return self.numerator == other.numerator and self.denominator == other.denominator
+
     def __str__(self) -> str:
-        if self.numerator != 0:
-            return str(self.denominator / self.numerator)
+        if self.denominator != 0:
+            return str(self.numerator / self.denominator)
         else:
             return "0.0"
 
-    def __eq__(self, other: FrameInterval) -> bool:
-        return self.numerator == other.numerator and self.denominator == other.denominator
+    def as_float(self) -> float:
+        if self.denominator != 0:
+            return self.numerator / self.denominator
+        else:
+            return 0.0
+
+    def to_frame_rate(self) -> float:
+        if self.numerator != 0:
+            return self.denominator / self.numerator
+        else:
+            return 0.0
```

Comment:
I don't think this is needed, users can still calculate the fps easily (`fps = 1/frame_interval`).

I do however see benefit in having a quick access to this, but for this, but I'd rather have a dedicated `get_fps` function in `class Device`, I think it should be fine if it just returns a float, but having a dedicated `FrameRate` class would also work (it's probably not necessary though). This is optional, feel free to just remove this function if that's what you prefer.